cft/deploy.yml

```diff
@@ -0,0 +1,38 @@
+Resources:
+  ElasticsearchDomain:
+    Type: AWS::Elasticsearch::Domain
+    Properties:
+      DomainName: "test"
+      ElasticsearchClusterConfig:
+        DedicatedMasterEnabled: true
+        InstanceCount: 2
+        ZoneAwarenessEnabled: true
+        InstanceType: "m3.medium.elasticsearch"
+        DedicatedMasterType: "m3.medium.elasticsearch"
+        DedicatedMasterCount: 3
+      EncryptionAtRestOptions:
+        Enabled: false
+      EBSOptions:
+        EBSEnabled: true
+        Iops: 0
+        VolumeSize: 20
+        VolumeType: "gp2"
+      SnapshotOptions:
+        AutomatedSnapshotStartHour: 0
```

Comment:
Coverity Rapid Scan found issue: Missing TLS - CWE-319, Medium Severity

**The Amazon Elasticsearch Service (Amazon ES) domain does not use TLS for node-to-node encryption within the VPC.
**

**How to fix:** Enable node-to-node encryption by explicitly setting the `Properties.NodeToNodeEncryptionOptions.Enabled` value to `true`.